libs/platform/src/lib/components/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.html

```diff
@@ -0,0 +1,53 @@
+<div
+    class="fd-dynamic-page__collapsible-header"
+    [attr.aria-label]="headerAriaLabel"
+    [attr.aria-hidden]="collapsed"
+    [attr.id]="id"
+    #headerContent
+>
+    <ng-content></ng-content>
+</div>
+<div
+    class="fd-dynamic-page__collapsible-header-visibility-container"
+    style="min-height: 0.0625rem;"
+    #pincollapseContainer
+>
+    <ng-container *ngIf="collapsible">
+        <div class="fd-dynamic-page__collapsible-header-visibility-container--left-gradient"></div>
+        <div class="fd-dynamic-page__collapsible-header-visibility-container--button-group">
+            <button
+                class="fd-dynamic-page__collapse-button fd-button fd-button--compact"
+                [attr.aria-label]="expandCollapseAriaLabel"
+                [attr.title]="expandCollapseAriaLabel"
+                [attr.aria-expanded]="!collapsed"
+                aria-haspopup="true"
+                [attr.aria-controls]="id"
+                *ngIf="collapsible && !pinned"
+                (click)="toggleCollapse()"
+            >
+                <i *ngIf="!collapsed" class="sap-icon--slim-arrow-up"></i>
+                <i *ngIf="collapsed" class="sap-icon--slim-arrow-down"></i>
+            </button>
+            <button
+                class="fd-dynamic-page__pin-button fd-button fd-button--compact"
+                [attr.aria-label]="pinUnpinAriaLabel"
+                [attr.title]="pinUnpinAriaLabel"
+                [attr.aria-selected]="pinned"
+                (click)="onPinned()"
+                *ngIf="pinnable"
+            >
+                <i class="sap-icon--pushpin-off"></i>
+            </button>
+            <!-- <fdp-button
+            [buttonType]="'emphasized'"
+            [glyph]="'pushpin-off'"
+            title="Pinnable"
+            contentDensity="compact"
+            [ariaSelected]="pinnable"
```

Comment:
remove commented code

Reply:
👍 will change it to fdp-button/fd-button usage, was waiting for the glyph issue to get resolved.